Try to reproduce illegal instruction on CI

jorisv opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Close #14

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ The recipe is not parsable by parser `conda-souschef (grayskull)`. This parser is not currently used by conda-forge, but may be in the future. We are collecting information to see which recipes are compatible with grayskull.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/19359392373](https://github.com/conda-forge/conda-forge-webservices/actions/runs/19359392373). Examine the logs at this URL for more detail.</sub>

ManifoldFR commented on recipe/conda_build_config.yaml:
```diff
@@ -1,7 +1,5 @@
-MACOSX_SDK_VERSION:         # [osx and x86_64]
-  - "10.15"                 # [osx and x86_64]
+# Workaround for https://github.com/conda-forge/opencv-feedstock/issues/493
+MACOSX_SDK_VERSION:        # [osx]
+  - "15.5"                 # [osx]
```
interesting

jorisv commented:
Ok, we found a fix \o/ thanks to https://github.com/conda-forge/opencv-feedstock/issues/493 and @olivier-roussel.
But I will wait the 0.13 release to merge it.

jorisv committed:
Test if tracy-profiler run

jorisv committed:
Bump build number

jorisv committed:
MNT: Re-rendered with conda-smithy 3.53.0 and conda-forge-pinning 2025.11.14.03.49.41

Other tools:
- conda-build 25.9.0
- rattler-build 0.49.0
- rattler-build-conda-compat 1.4.9

jorisv commented:
Since 0.13 is blocked, I will merge it right now.
